app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<h1>Task List</h1>
+
+<table>
+  <tr>
+    <th>
+      Name
+    </th>
+    <th>
+      Owner
+    </th>
+    <th>
+      Status
+    </th>
+    <th class="center">
+      Mark Done
+    </th>
+    <th class="center">
+      Details
+    </th>
+    <th class="center">
+      Delete
+    </th>
+  </tr>
+  <tbody>
+    <% @all_tasks.each do |task| %>
+      <% if task.completed_date.present? %>
+        <% @task_css = "completed" %>
```

Comment:
I was referencing your earlier comment regarding `app/controllers/tasks_controller.rb` `line 9`, "to have a Task#completed? method to encapsulate the logic of whether or not the task is completed in the model." So if it's setting a CSS variable specifically, that logic should instead be in a view helper?